PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/ui/component/appservice/AppExistingComponent.kt

```diff
@@ -106,7 +107,7 @@ abstract class AppExistingComponent<TApp : WebAppBase, TSlot: DeploymentSlotBase
 
         val osColumnIndex = (pnlExistingAppTable.table.model as DefaultTableModel).findColumn(pnlExistingAppTable.appTableColumnOs)
         require(osColumnIndex >= 0) { "Column index is out of range" }
-        sorter.rowFilter = regexFilter(OperatingSystem.WINDOWS.name.toLowerCase().capitalize(), osColumnIndex)
+        sorter.rowFilter = regexFilter(OperatingSystem.WINDOWS.name.lowercase().replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.getDefault()) else it.toString() }, osColumnIndex)
```

Comment:
Is that for Linux?

Reply:
Suggested after updating Kotlin version, old method was deprecated